### ISSUE TITLE
need 1030000 iteration? and how about training time?

### ISSUE THREAD

**Issue by zilesazoyi2** [eldar/pose-tensorflow#29]
# Thanks your code !really helpful! so great!

my laptop is GTX960 and I worry about the training time and whether GTX960 can afford?
It seems like need 1030000 iteration, can I  cut down to 100000?
 after 60000 iters ,the loss is 0.0410 and my laptop temperature nearly 90 C.
 could you give some advice?
thanks for you kindness
# Learning rate schedule for the SGD optimiser. 
multi_step:
- [0.005, 10000]
- [0.02, 430000]
- [0.002, 730000]
- [0.001, 1030000]

my  configuration 
laptop gtx960 ,
memory
4G

**Comment by eldar** [comment]
This number of training iterations worked best for me, but one can experiment and try better hyper-parameters. Closing because of the lack of activity.
